redbot/core/core_commands.py

```diff
@@ -1557,8 +1557,22 @@ async def set_showsettings(self, ctx: commands.Context):
             mod_role_ids = guild_data["mod_role"]
             mod_role_names = [r.name for r in guild.roles if r.id in mod_role_ids]
             mod_roles_str = humanize_list(mod_role_names) if mod_role_names else "Not Set."
-            guild_settings = _("Admin roles: {admin}\nMod roles: {mod}\n").format(
-                admin=admin_roles_str, mod=mod_roles_str
+
+            guild_locale = await self.bot._i18n_cache.get_locale(ctx.guild)
+            guild_regional_format = await self.bot._i18n_cache.get_regional_format(ctx.guild) or _(
+                "Same as bot's locale"
```

Comment:
Might I suggest putting the bot's locale in brackets so guild admins know what the setting is?

Reply:
_pst_ and maybe add a lil dot on the end of the sentence as well.

Reply:
a dot doesn't make sense here.

Reply:
Resolved in https://github.com/Cog-Creators/Red-DiscordBot/pull/3896/commits/9f07ae0b7e17f5e2dfaacb9d0ad8c58454d2d7bf